app.js

```diff
@@ -10,7 +10,6 @@ function deletePet(event) {
     event.target.parentElement.parentElement.remove()
     const id = findId(event);
     pets = pets.filter(pet => pet.id !== parseInt(id));
-    console.log(pets)
```

Comment:
@the-beatle ¿por qué agregaste este comentario?

Reply:
Hola, el comentario se agrega para explicar el concepto

Reply:
Por que agregaste ese comentario?